featuregate/Makefile

```diff
@@ -1,7 +1,7 @@
 # Copyright 2021 VMware, Inc. All Rights Reserved.
```

Comment:
Should this top-level directory be named `featuregates` or `features` instead of `featuregate`?

Reply:
I'm inclined to use `featuregates`, since it's not a package name, I'm okay with plural